buildchain/buildchain/config.py

```diff
@@ -63,6 +63,7 @@ class ExtCommand(enum.Enum):
     VAGRANT = os.getenv("VAGRANT_BIN", "vagrant")
     OPERATOR_SDK = os.getenv("OPERATOR_SDK_BIN", "operator-sdk")
     GOFMT = os.getenv("GOFMT_BIN", "gofmt")
+    TOX = os.getenv("TOX_BIN", "tox")
```

Comment:
If you wonder why I added this check, I started by using `tox` for rendering the rules manifest, but it was strange to add a requirement for both `tox` and `poetry` to be available for building the product, just for this small manifest. So I went with Docker instead :roll_eyes: